connectors/utils.py

```diff
@@ -35,7 +35,8 @@
 DEFAULT_CHUNK_MEM_SIZE = 25
 DEFAULT_MAX_CONCURRENCY = 5
 DEFAULT_CONCURRENT_DOWNLOADS = 10
-DEFAULT_BULK_MAX_RETRIES = 3
+DEFAULT_BULK_MAX_RETRIES = 5
+DEFAULT_BULK_RETRY_INTERVAL = 10
```

Comment:
Should these be imported from config.py?

Reply:
Ah these are left after me refactoring things, I'll delete these, thanks!

Reply:
Addressed in 244ce2e7e29534bf2c08507e5fa59f496524e228